pkg/clients/kubernetes.go

```diff
@@ -184,12 +184,6 @@ func GetSecretData(secretName, namespace string) (map[string][]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-	secretData, err := k8scli.Client.CoreV1().Secrets(namespace).Get(secretName, metav1.GetOptions{})
-	if err != nil {
-		log.Errorf("Unable to load secret '%s' from namespace '%s'", secretName, namespace)
-		return make(map[string][]byte), nil
-	}
-	log.Debugf("Found secret with name %v\n", secretName)
 
-	return secretData.Data, nil
+	return k8scli.GetSecretData(secretName, namespace)
```

Comment:
As you mentioned in IRC it is strange to have 2 of these functions. Think its worth getting rid of the duplication as part of a separate issue?

Reply:
@maleck13 yeah I think a separate PR will be better to fix this duplication. I can do that.